data/yam/agama/auto/sles_sap_default_ppc64le.jsonnet

```diff
@@ -12,7 +12,19 @@
   root: {
     password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
     hashedPassword: true,
+    sshPublicKey: 'enable ssh',
```

Comment:
value sound strange for something called `sshPublicKey`. How it works? What is the idea behind it?

Reply:
Clearly explained in the PR description. Thanks

Reply:
I notice somebody use real pubkey: https://github.com/lemon-suse/os-autoinst-distri-opensuse/blob/1bacd28fc716ddad516e23a9c928a6ff11bfe10b/data/yam/agama/auto/sles_sap_default.jsonnet#L14C5-L14C17

Reply:
please refer to https://bugzilla.suse.com/show_bug.cgi?id=1238590
YAM colleague gave me an example: https://openqa.suse.de/tests/17012526, adding `sshPublicKey` then agama is able to set sshd service enabled.